db/sql/0_msar.sql

```diff
@@ -101,10 +101,22 @@ $$ LANGUAGE plpgsql RETURNS NULL ON NULL INPUT;
 ----------------------------------------------------------------------------------------------------
 -- INFO FUNCTIONS
 --
--- Functions in this section get information about a given table or column.
+-- Functions in this section get information about a given schema, table or column.
 ----------------------------------------------------------------------------------------------------
 ----------------------------------------------------------------------------------------------------
 
+CREATE OR REPLACE FUNCTION __msar.get_sch_name(sch_id oid) RETURNS TEXT AS $$/*
+Return the name for a given schema, quoted as appropriate.
```

Comment:
Add a little more here; similar to the `get_relation_name` docstring.

Reply:
You don't need "qualified", just note the variable and describe its meaning under `Args:`.